### PR TITLE
new signup page

### PR DESCRIPTION
**Pull request by talhasch**

### CODE REVIEW

File: src/common/i18n/locales/en-US.json [hunk 1]
```diff
@@ -106,8 +106,10 @@
     "sign-up-text-2": "Sign up now!"
   },
   "sign-up": {
-    "header": "Join",
-    "description": "<h3>Home of Free speech and Rewarding communities!</h3><br>One account to manage everything on <b>Blockchain</b> and <b>Ecency</b>.<br><br>Learn more about benefits in our <a href=\"/faq#how-to-join\">FAQ</a>",
+    "header": "Welcome to Ecency",
+    "description": "Home of Free speech and Rewarding communities!",
+    "bottom-description": "Every account is created on a blockchain and only you have control over that account. One to manage everything on Blockchain and Ecency.",
```
Comment (feruzm) [inline src/common/i18n/locales/en-US.json:111]: Word dropped here: `One **account** to manage everything on Blockchain.`

**Comment by feruzm** [comment]
@talhasch also let's change ratio to 50/50. Image half of the screen not more and form other half.

**Comment by talhasch** [comment]
@feruzm done

**Comment by feruzm** [comment]
@talhasch looking great, one small thing... can we decrease this space a bit?
image

**Comment by talhasch** [comment]
ok. decreased from 120px to 60px

**Review by feruzm — APPROVED** [review]
lgtm 🚀

**Pull request merged**